### PR TITLE
Registration page and global styles

### PR DESCRIPTION
**Pull request by ladyofcode**

### CODE REVIEW

File: frontend/src/pages/Registration/Registration.styles.tsx [hunk 1]
```diff
@@ -1 +1,79 @@
...
+
+    background: rgb(254,254,254);
+    background: linear-gradient(0deg, rgba(254,254,254,1) 0%, rgba(203,218,239,1) 51%, rgba(254,254,254,1) 100%);
+    text-align: left;
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    color: #1E2354;
```
Comment (donnyroufs) [inline frontend/src/pages/Registration/Registration.styles.tsx:16]: Incase we are re-using the colours we can move it to theme.ts

File: frontend/src/pages/Registration/Registration.styles.tsx [hunk 2]
```diff
@@ -1 +1,79 @@
...
+
+        button {
+            align-self: center;
+            padding: 10px 20px;
+            width: 150px;
+            background: rgb(203,218,239);
+            background: linear-gradient(0deg, rgba(203,218,239,1) 51%, rgba(254,254,254,1) 100%); 
+            font-weight: bold;
+            color: #1E2354;
+            border: 1px #d4d4d4;
```
Comment (donnyroufs) [inline frontend/src/pages/Registration/Registration.styles.tsx:71]: Incase we are re-using the colours we can move it to theme.ts

File: frontend/src/pages/Registration/Registration.styles.tsx (hunk 1 above)
Comment (ladyofcode) [reply frontend/src/pages/Registration/Registration.styles.tsx:16]: kk I will do that

File: frontend/src/pages/Registration/Registration.tsx [hunk 3]
```diff
@@ -26,24 +30,34 @@ export default class Registration extends React.Component {
...
+                    <span>
+                        <label>Status</label>
+                        <select id="status" name="status">
+                            <option value="online">Online</option>
+                            <option value="offline">Offline</option>
+                            <option value="dnd">Do not disturb</option>
+                        </select>
+                    </span>
+                    
+                    <span>
```
Comment (donnyroufs) [inline frontend/src/pages/Registration/Registration.tsx:42]: Technically this should be a div :P but its fine. Lets keep the hackathon spirit

**Pull request merged**